plugins/render/api_key_test.go

```diff
@@ -0,0 +1,37 @@
+package render
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.APIKey: "rnd_Z7xMKp4NX1FoQNRyBpZs9yxDbu3i",
+			},
+		},
+	})
+}
+
+
+func TestAPIKeyImporter(t *testing.T) {
+	plugintest.TestImporter(t, APIKey().Importer, map[string]plugintest.ImportCase{
+		"config file": {
+			Files: map[string]string{
+				"~/.render/config.yaml": plugintest.LoadFixture(t, "config.yaml"),
```

Comment:
There looks to be no fixture checked into source control.

Let's make sure we add that, and that the tests pass locally as well (`make test`).

I'll trigger the pipeline to make sure they pass in there as well.

Reply:
I have added a fixture to it.